### PR TITLE
Added RESTAPI policies check #43

### PR DESCRIPTION
**Pull request by leandrodamascena**

### CODE REVIEW

File: clouddiscovery/provider/vpc/resource/network.py [hunk 1]
```diff
@@ -583,3 +586,62 @@ def get_resources(self) -> List[Resource]:
...
+        resources_found: List[Resource] = []
+
+        # get REST API available
+        response = client.get_rest_apis()
+
+        message_handler("Collecting data from REST API Policies...", "HEADER")
+
+        with ThreadPoolExecutor(15) as executor:
+            results = executor.map(
+                lambda data: self.analyze_restapi(data), response["items"]
```
Comment (meshuga) [inline clouddiscovery/provider/vpc/resource/network.py:615]: Codacy Issue found: [Lambda may not be necessary](https://app.codacy.com/gh/Cloud-Architects/cloudiscovery/pullRequest?prid=5694158)

**Comment by meshuga** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Cloud-Architects/cloudiscovery/pullRequest?prid=5694158&bid=18635708)

File: clouddiscovery/provider/vpc/resource/network.py (hunk 1 above)
Comment (leandrodamascena) [reply clouddiscovery/provider/vpc/resource/network.py:615]: Ok, makes sense.. I just copied from S3.

Comment (leandrodamascena) [reply clouddiscovery/provider/vpc/resource/network.py]: Check now.

**Review by patrykorwat — APPROVED** [review]

**Pull request merged**